tests/schema/test_windows_paths.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pathlib
+
+from siliconcompiler import Chip
+
+
+def test_windows_path_compat():
```

Comment:
spell out compatibility
also this wasn't where the failure occurred, we should have a test ensure that is also working